libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -2453,6 +2463,31 @@ bool AP_Mission::distance_to_landing(uint16_t index, float &tot_distance, Locati
     return ret;
 }
 
+// Calculate the radial distance to mission leg, in the plane normal to the mission leg.
+// The end points of the mission leg are observed, so the error will increase more than
+// the radius beyond the end of the mission leg.
+bool AP_Mission::calc_norm_radius_to_mission_leg(const Location last_wp_loc, const Location next_wp_loc, const Location current_loc, float &radius) const
```

Comment:
This is not what SCurves will try and fly.

Reply:
Agreed.  The purpose of this method was to give a simple and clear measure of deviation from the intended route.  I believe this method is suitable (and most appropriate) to what is needed for this landing improvement.  However, this is an area that we should revisit before this applied else where in the mission, hence I do not think that this PR should be used beyond the scope of landings for now.